js/test/daterange/testdatefmtrange_fmttemplate.js

```diff
@@ -13936,6 +13936,81 @@ module.exports.testdaterangefmt = {
         test.equal(templatec30[2], "{sy} – {ey}");
         test.equal(templatec30[3], "{sy} – {ey}");
 
+        test.done();
+    },
+    testDateRngFmt_my_MM: function(test) {
+        test.expect(36);
+        var fmt;
+        var data = setVariables();
+        var length = data["fullLength"];
+        var abbrLength = data["abbrLength"];
+
+        var templatec00 = [],templatec01 = [],templatec02 = [],templatec03 = [];
+        var templatec10 = [],templatec11 = [],templatec12 = [],templatec20 = [];
+        var templatec30 = [];
+
+        // full, medium: ymd
+        // long: ydm
+        // short: dmy
+        
+        for (var i=0; i < 4; i++) {
+            fmt = new DateRngFmt({locale:"my-MM", length: length[i]})
+            templatec00.push(fmt.dateFmt.formats.range["c00"][abbrLength[i]]);
+            templatec01.push(fmt.dateFmt.formats.range["c01"][abbrLength[i]]);
+            templatec02.push(fmt.dateFmt.formats.range["c02"][abbrLength[i]]);
+            templatec03.push(fmt.dateFmt.formats.range["c03"][abbrLength[i]]);
+            templatec10.push(fmt.dateFmt.formats.range["c10"][abbrLength[i]]);
+            templatec11.push(fmt.dateFmt.formats.range["c11"][abbrLength[i]]);
+            templatec12.push(fmt.dateFmt.formats.range["c12"][abbrLength[i]]);
+            templatec20.push(fmt.dateFmt.formats.range["c20"][abbrLength[i]]);
+            templatec30.push(fmt.dateFmt.formats.range["c30"][abbrLength[i]]);
+        }
+
+        test.equal(templatec00[0], '{sy}၊ {sm} {sd} {st} – {et}');
+        test.equal(templatec00[1], '{sy}၊ {sd} {sm} {st} – {et}');
+        test.equal(templatec00[2], '{sy}၊ {sm} {sd} {st} – {et}');
+        test.equal(templatec00[3], '{sd}-{sm}-{sy} {st} – {et}');
```

Comment:
I'll fix locale data manually in this PR first, then will fix the tool code to convert properly.

Reply:
Regarding a `short` length dateformat in CLDR, it is `"short": "dd-MM-yy"` it will be replaced the number not their character. so I think it doesn't need to have "၊" character. no?

Reply:
Other formats that have day-month-year are followed by that character, so this should be consistent